lib/krb5/get_addrs.c

```diff
@@ -50,6 +50,14 @@ gethostname_fallback (krb5_context context, krb5_addresses *res)
     char hostname[MAXHOSTNAMELEN];
     struct hostent *hostent;
 
+    if (krb5_config_get_bool(context, NULL, "libdefaults", "block_dns",
```

Comment:
Is this needed here?  I expect in many cases there will be an `/etc/hosts` entry for the local host's name.